### ISSUE TITLE
[Halo] All BSPs are loaded in a map, leading to perf issues and z-fighting

### ISSUE THREAD

**Issue by notfrying1pans** [magcius/noclip.website#572]
For an example on the z-fighting, I believe thesea re the pre- and post-flood reveal bsps rendering at the same time in 343 Guilty Spark 

![image](https://user-images.githubusercontent.com/49667761/216701040-801e51d5-c14b-46f4-a077-e5cd0f9d0aac.png)

**Comment by magcius** [comment]
Yeah, this was a concern when we were adding them. We could add a Layers panel on the left like we do in some other games which lets you toggle between BSPs.
